js/src/product-feed/review-request/index.js

```diff
@@ -20,6 +20,7 @@ const showNotice = ( status ) => !! REVIEW_STATUSES[ status ]?.title;
 const ReviewRequest = ( { account = {} } ) => {
```

Comment:
💅 
It would be nice to add JSDoc for the events and their context/action values. Therefore, the tracking docs would have up-to-date docs about them when auto-generating.
- gla_modal_open
   - `context: 'request-review'`
- gla_modal_closed
   - `context: 'request-review'`
   - `action: 'request-review-success' | 'maybe-later' | 'dismiss'`